solutions/two_sum_solution.rb

```diff
@@ -1,4 +1,15 @@
 class Array
+  # Define a method, `two_sum`, that finds all 
```

Comment:
Should be `Array#two_sum` or otherwise specify that it's monkey patched onto the Array class